deploy/gcloud/deploy.sh

```diff
@@ -0,0 +1,107 @@
+# This shell script is used by the docker init container used in the
```

Comment:
This file seems to have a lot in common with gcloud_up.sh. Is it possible to extract the commonalities into a shared .sh?

Reply:
Yes, the idea is to eventually remove the similar calls in gcloud_up.sh and use helm combined with the init container to do the container initialization. It's on my todo list but leaving the gcloud* scripts alone for now until click-to-deploy works end-to-end.

Reply:
Ah, ok, SG.